client/css/style.css

```diff
@@ -36,6 +38,16 @@
 	font-style: normal;
 }
 
+html {
+	box-sizing: border-box;
+}
+
+*,
+*:before,
+*:after {
+	box-sizing: inherit;
```

Comment:
I can't find the comment that this refers to so I'm guessing it was: Why do we need this here?!
This seems like an addition that was necessary after the normalize.css bump, but since it was brought back to 3.0.1, these lines should not be necessary anymore.